src/case-studies/mosaic/MosaicCaseStudy.module.css

```
@@ -0,0 +1,982 @@
+    -apple-system,
+    BlinkMacSystemFont,
+    'Segoe UI',
+    sans-serif;
+  font-size: 0.95rem;
+  font-weight: 700;
+  letter-spacing: 0;
+  line-height: 1.35;
+}
+
+.typeRole[data-role='heading'] strong {
+  font-family: 'Fraunces', Georgia, serif;
```

Stylelint (17.14.0)

[error] Expected no quotes around "Fraunces" (font-family-name-quotes)

(font-family-name-quotes)